parent/pom.xml

```
@@ -244,7 +244,7 @@
         <plugin>
           <groupId>org.codehaus.cargo</groupId>
           <artifactId>cargo-maven3-plugin</artifactId>
-          <version>1.10.16</version>
+          <version>1.10.20</version>
```

```suggestion
          <version>1.10.21</version>
```
Wait till this version is released and change the qa container ID to `wildfly37`

---

Follow up issue: https://github.com/camunda/camunda-bpm-platform/issues/5333